src/main/java/ch/njol/skript/expressions/ExprInventoryCloseReason.java

```diff
@@ -57,28 +57,14 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 		}
 		return true;
 	}
-	
-	@Override
-	protected InventoryCloseEvent.Reason[] get(Event event) {
-		if (!(event instanceof InventoryCloseEvent))
-			return null;
 
-		return new InventoryCloseEvent.Reason[]{((InventoryCloseEvent) event).getReason()};
+	public ExprInventoryCloseReason() {
+		super(InventoryCloseEvent.Reason.class);
 	}
-	
-	@Override
-	public boolean isSingle() {
-		return true;
-	}
-	
-	@Override
-	public Class<? extends InventoryCloseEvent.Reason> getReturnType() {
-		return InventoryCloseEvent.Reason.class;
-	}
-	
+
 	@Override
 	public String toString(@Nullable Event event, boolean debug) {
-		return "inventory close reason";
+		return "the inventory close reason";
```

Comment:
Doesn't need the word `the`

Reply:
can you suggest changes, I am away of my laptop

Reply:
Although it doesn't really matter, even teleport cause uses it

Reply:
No, it shouldn't be added, because if the user uses the word `the` the debug logs can say `the the inventory close reason` it uses the user's input + the expression.toString which is why we don't add the or a into toStrings

Reply:
Good to know 👍

Reply:
```suggestion
		return "inventory close reason";
```